docs/general/ddl/generated-columns.rst

```diff
@@ -31,8 +31,8 @@ expression if omitted::
 
    For a full syntax description, see :ref:`sql-create-table`.
 
-Generated columns are read-only. Their values are computed as needed for every
-``INSERT`` and ``UPDATE`` operation.
+The values of generated columns are computed as needed for every ``INSERT``,
+``UPDATE`` and ``COPY FROM`` operation.
 
```

Comment:
In PG users can't provide a value for a generated column, 
see https://www.postgresql.org/docs/17/ddl-generated-columns.html

>A generated column cannot be written to directly. In INSERT or UPDATE commands, a value cannot be specified for a generated column

Since CrateDB allow passing a value (but only for validation), shouldn't we mention it here?

From PR description https://github.com/crate/crate/pull/18323#issue-3381844487

>generated: insert: Generate if absent; validate if present, unless nondeterministic - in that case use as-is
generated: update: Re-generate if absent; validate if present, unless nondeterministic - in that case use as-is.

Reply:
>but only for validation
actually, non-determinsitc provided value will be used as is, so users can write any custom value to a column looking like `a int as random()`.

Reply:
We could add an extra note there that for non-deterministic the value is not validated, e.g.:
```
cr> create table t2(a int, b generated always as  random());
CREATE OK, 1 row affected (0.404 sec)
cr> insert into t2(a) values(1);
INSERT OK, 1 row affected (0.040 sec)
cr> insert into t2(a,b ) values(1,3);
INSERT OK, 1 row affected (0.020 sec)
cr> select * from t2;
+---+--------------------+
| a |                  b |
+---+--------------------+
| 1 | 3.0                |
| 1 | 0.2576447529090903 |
+---+--------------------+
SELECT 2 rows in set (0.005 sec)
```

Reply:
LGTM, thanks!